tests/v3/covid-19/api_gov.spec.js

```diff
@@ -19,7 +19,8 @@ const countries = [
 	'New Zealand',
 	'Colombia',
 	'South Africa',
-	'UK'
+	'UK',
+	'Israel'
```

Comment:
Can you adjust this file to only fix the v2-->v3 route testing bug, no Israel stuff